website/www/site/content/en/documentation/io/built-in/google-bigquery.md

```diff
@@ -981,6 +981,14 @@ BigQueryIO currently has the following limitations.
    multiple BigQuery tables. The Beam SDK for Java does not have this limitation
    as it partitions your dataset for you.
 
+3. When you [load data](https://cloud.google.com/bigquery/docs/loading-data) into BigQuery, [these limits](https://cloud.google.com/bigquery/quotas#load_jobs) are applied.
+By default, BigQuery uses a shared pool of slots to load data.
```

Comment:
shared pool for the GCP project I assume?

Reply:
free pool of slots is shared among multiple customers